src/main/java/com/github/pop4959/srbot/command/CommandMatchmaker.java

```diff
@@ -55,10 +66,11 @@ private void findPlayersForLobby(Set<Member> lobby, List<Member> pool) {
         Collections.shuffle(pool);
         for (Member member : pool) {
             Game game = member.getGame();
-            if (game != null && "SpeedRunners".equals(game.getName())) {
+            if (game != null && game.getName().equals("SpeedRunners")) {
```

Comment:
Please revert this because it could cause a null pointer exception when game.getName() is null. The code was written as it was before on purpose.